charts/agent/values.yaml

```diff
@@ -73,9 +70,31 @@ node:
       lookbackPeriod: 24h
       # At startup, where to start reading logs from the file. Options are beginning or end.
       startAt: end
-
+    # pulls node, pod, and container metrics from the API server on a kubelet and sends it down the metric pipeline for further processing.
     metrics:
       enabled: true
+      interval: 60s
+
+application:
+  # use this option to scrape prometheus metrics from pods
+  # !!! IMPORTANT !!! cluster.metrics.enabled must be set to true
```

Comment:
can we just do an enabled field here too and then make the if statement `cluster.metrics.enabled or application.enabled`?

Reply:
Helm chart doesn't sort this logic and the daemon set is conditional on cluster.metrics.enabled

I think for now we live with it until we find a better way